### PR TITLE
just-zip

### PR DESCRIPTION
**Pull request by kamilogorek**
Adds `just-zip` function and fixes some minor lint warnings.

Related: https://github.com/angus-c/just/issues/36

### CODE REVIEW

File: test/function-debounce/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ test('waits for n ms then runs once', function (t) {
   t.plan(1);
   var callCounter = 0;
   var fn = debounce(
-    function () {callCounter++},
+    function () {callCounter++;},
```
Comment (angus-c) [inline test/function-debounce/index.js]: we're not adding semicolons for single line functions

File: test/function-debounce/index.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ test('when callFirst is true, runs once, waits for n ms then runs again', functi
   t.plan(2);
   var callCounter = 0;
   var fn = debounce(
-    function () {callCounter++},
+    function () {callCounter++;},
```
Comment (angus-c) [inline test/function-debounce/index.js]: as above

File: test/function-debounce/index.js (hunk 1 above)
Comment (kamilogorek) [inline test/function-debounce/index.js]: Got it. I changed it, as eslint was pointing it as a warning.

**Comment by kamilogorek** [comment]
Added more examples (used all test cases) and updated README with an appropriate entry.

**Comment by angus-c** [comment]
thanks!

**Pull request merged**